recipe/vs2010_omp.patch

```diff
@@ -0,0 +1,12 @@
+--- Source/LibRawLite/libraw/libraw_types.h
++++ Source/LibRawLite/libraw/libraw_types.h
+@@ -32,7 +32,7 @@
+ #if defined (_OPENMP)
+ 
+ #if defined(WIN32)
+-# if defined (_MSC_VER) && (_MSC_VER >= 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
++# if defined (_MSC_VER) && (_MSC_VER > 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
```

Comment:
Oh, apparently I did change something to that patch. I must have made that change to make it work for my build system. Anyway, the patch itself was already there.